package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "license": "UNLICENSED",
   "scripts": {
-    "private::format.black": "black dash tests --exclude metadata_test.py",
+    "private::format.black": "black dash tests components --exclude metadata_test.py",
```

Comment:
That's a bit over-broad, and anyway both dcc and table already have their own linters and formatters. But since html doesn't, if you want to include just html here go ahead - just please also add it to the lint commands for black, flake8, and pylint.

Reply:
I think I added that for convenience because it's linted somewhere in ci but running `npm run format` from root didn't format the tests in dcc.

Reply:
I see - then the right way to handle this is to make new commands for each component package that mirror the renderer, ie they call out to the format commands defined in those packages:

https://github.com/plotly/dash/blob/4b03e5ac9e6406a4fd35f8b7b2fde33663959885/package.json#L6

And if these new commands fit the `private::format.*` pattern they'll be picked up by the root `npm run format` command